### PR TITLE
Introduce Reactive Principles

### PR DESCRIPTION
**Pull request by ignasi35**
I did a first pass at cross-linking the reactive principles from the existing Concepts and the stubbed pages. I'm not sure we need to create an extra page to present the principles (it'd prbably be a duplicate of the https://principles.reactive.foundation/principles/) 

References #211

Did some random cleanup in the process.

### CODE REVIEW

File: docs-source/docs/modules/concepts/pages/introduction-traditional-architecture.adoc [hunk 1]
```diff
@@ -1,115 +0,0 @@
-= Introduction: Traditional architectures versus Reactive Microsystems
```
Comment (ignasi35) [inline docs-source/docs/modules/concepts/pages/introduction-traditional-architecture.adoc:1]: I moved the content of this page into `concepts/pages/index.adoc` long ago but forgot to remove this.

File: docs-source/docs/modules/concepts/pages/autonomy.adoc [hunk 2]
```diff
@@ -14,3 +14,6 @@ Another aspect of autonomy is that if a service only can make promises about its
 Working with autonomous services opens up flexibility around service orchestration, workflow management and collaborative behavior, as well as scalability, availability and runtime management, at the cost of putting more thought into well-defined and composable APIs that can make communication—and consensus—a bit more challenging—something we will discuss shortly.
 
 include::partial$reactive-microservices-ebook.adoc[]
+
+NOTE: Autonomy is one oif the {reactive-principles}/principles/assert-autonomy.html[Reactive Principles] {reactive-principles}/principles/assert-autonomy.html[{tab-icon}, window="tab"].
```
Comment (ignasi35) [inline docs-source/docs/modules/concepts/pages/autonomy.adoc:18]:
```suggestion
NOTE: Autonomy is one of the {reactive-principles}/principles/assert-autonomy.html[Reactive Principles] {reactive-principles}/principles/assert-autonomy.html[{tab-icon}, window="tab"].
```

File: docs-source/docs/modules/concepts/pages/circuit-breakers.adoc [hunk 3]
```diff
@@ -13,15 +13,14 @@ remote systems to prevent the failure of a single component from bringing down a
 As an example, we have a web application interacting with a remote third party web service.  
 Let's say the third party has oversold their capacity, and their database melts down under load.
 Assume that the database fails in such a way that it takes a very long time to hand back an
-error to the third party web service.  This in turn makes calls fail after a long period of 
-time.  Back to our web application, the users have noticed that their form submissions take
```
Comment (ignasi35) [inline docs-source/docs/modules/concepts/pages/circuit-breakers.adoc:17]: `s/a long period of time/period` as suggested by intellisense.

File: docs-source/docs/modules/concepts/pages/cqrs.adoc [hunk 4]
```diff
@@ -27,7 +27,9 @@ You can start refactoring an application towards a CQRS architecture by identify
 
 As described above, CQRS doesn't require the write-side handling the commands to be implemented using xref:event-sourcing.adoc[Event Sourcing]. Using Event Sourcing, though, is a perfect fit for CQRS.
 
-In CQRS, in order to build the read-side representations we want a stream of data changes to build projections from, and that is exactly what Event Sourcing provides. In Event Sourcing, state is stored as as an append-only, ordered list of changes (referred to as events).
+In CQRS, in order to build the read-side representations we want a stream of data changes to build projections from, and that is exactly what Event Sourcing provides. In Event Sourcing, state is stored as an append-only, ordered list of changes (referred to as events).
```
Comment (ignasi35) [inline docs-source/docs/modules/concepts/pages/cqrs.adoc:30]: `s/ as as/ as` (remove duplicate `as`)

File: docs-source/docs/modules/concepts/pages/eventual-consistency.adoc [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+:toc:
+:toc-title: ON THIS PAGE
+:toclevels: 2
+
+include::partial$include.adoc[]
+
+
+link to https://principles.reactive.foundation/principles/accept-uncertainty.html
+
+link to https://principles.reactive.foundation/principles/tailor-consistency.html
```
Comment (ignasi35) [inline docs-source/docs/modules/concepts/pages/eventual-consistency.adoc:11]: I think this concept was missing. It's referenced from multiple places and even relates to two of the principles.

**Review by patriknw — APPROVED** [review]
LGTM. Do we want to include more content from reactive principles by including summary of the most important concepts there? Is that already covered?
Anyway, this is good first step.

**Comment by ignasi35** [comment]
> LGTM. Do we want to include more content from reactive principles by including summary of the most important concepts there? Is that already covered?

I'm not sure. Yeah. On one hand I think an intro connecting all the Reactive-XYZ would be helpful: considering all possible values of XYZ:

- manifesto
- streams
- system
- principles
- patterns

I think a single page connecting them all is necessary.

Other than that, introducing the Reactive Principles should be something small to try to repeat too much from the actual site. 

In any case, it's content for another PR.

**Comment by ignasi35** [comment]
refs #255

**Pull request merged**